NEWS.md

```diff
@@ -6,6 +6,9 @@ main
  - Add regional simulation example PR [#757](https://github.com/CliMA/ClimaLand.jl/pull/757)
  - ![][badge-💥breaking] Extend photosynthesis mechanism parameter to support fields.
   PR[#774](https://github.com/CliMA/ClimaLand.jl/pull/774)
+ - ![][badge-💥breaking] Remove root_distribution from PlantHydraulicsParameters and add rooting_depth,
+ which supports fields
+  PR[#783](https://github.com/CliMA/ClimaLand.jl/pull/783)
```

Comment:
If something is breaking, it is always a good idea to describe how it is breaking and what users might need to do to fix it